### DISCUSSION TITLE
vim9script vs other embeddedable languages

### DISCUSSION THREAD

**Discussion by RayZ0rr** [vim/vim#13274] [Ideas]
This could probably be a very stupid writing but here goes.

Usually it's best practice to draw a rough blueprint or design before working on a project.

I'm pretty sure the vim team has done this when deciding to go with vim9script and starting it's development.

But I just want to confirm whether the team had considered other possible options such as the ones given here:

https://github.com/dbohdan/embedded-scripting-languages

**Comment by dkearns** [comment]
This is a bit of a hot topic in some circles but I'll take your query at face value. :)

Here's some initial discussion on Vim9 script - https://groups.google.com/g/vim_dev/c/__gARXMigYE/m/orGEJs3PBwAJ
